tools/c7n_gcp/tests/test_notify.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2015-2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from gcp_common import BaseTest
+
+
+class NotifyTest(BaseTest):
+
+    def test_notify_schema_validate(self):
+        factory = self.replay_flight_data('gcpnotifyvalidation')
```

Comment:
this test is extraneous given the same policy in the test below. tests run with validate on in ci. also you don't need a session recording for a validate.

Reply:
Removed in upcoming commit.